WARNING:root:Malformed experiment 'mlruns'. Detailed error Yaml file '/lustre/groups/imm01/workspace/irene.bonafonte/Projects/2023May_nichecompass/nichecompass-reproducibility/scripts/reference/mlruns/mlruns/meta.yaml' does not exist.
Traceback (most recent call last):
  File "/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/mlflow/store/tracking/file_store.py", line 270, in list_experiments
    experiment = self._get_experiment(exp_id, view_type)
  File "/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/mlflow/store/tracking/file_store.py", line 394, in _get_experiment
    meta = FileStore._read_yaml(experiment_dir, FileStore.META_DATA_FILE_NAME)
  File "/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/mlflow/store/tracking/file_store.py", line 1049, in _read_yaml
    return _read_helper(root, file_name, attempts_remaining=retries)
  File "/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/mlflow/store/tracking/file_store.py", line 1042, in _read_helper
    result = read_yaml(root, file_name)
  File "/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/mlflow/utils/file_utils.py", line 181, in read_yaml
    raise MissingConfigException("Yaml file '%s' does not exist." % file_path)
mlflow.exceptions.MissingConfigException: Yaml file '/lustre/groups/imm01/workspace/irene.bonafonte/Projects/2023May_nichecompass/nichecompass-reproducibility/scripts/reference/mlruns/mlruns/meta.yaml' does not exist.
/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/anndata/_core/anndata.py:1830: UserWarning: Observation names are not unique. To make them unique, call `.obs_names_make_unique`.
  utils.warn_names_duplicates("obs")
/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/umap/distances.py:1063: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/umap/distances.py:1071: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/umap/distances.py:1086: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/icb/irene.bonafonte/miniconda3/envs/nichecompass-reproducibility/lib/python3.9/site-packages/umap/umap_.py:660: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
